test/db/anal/mach0

```diff
@@ -7,37 +7,37 @@ pd 17 @0x8058
 EOF
 EXPECT=<<EOF
 |           0x00008058      .dword 0x0000e403
-            ; DATA XREF from sub.free_7418 @ 0x7d30
+            ; DATA XREF from sub.free_7890 @ 0x7d30
```

Comment:
do you know why this change?

Reply:
yes, the change is correct. well the old 7418 is wrong for sure.

Actually drawing the graph of this function spits out some warnings:

```
WARNING: r_agraph_add_edge: assertion 'g && a && b' failed (line 3823)
WARNING: r_agraph_add_edge: assertion 'g && a && b' failed (line 3823)
```

imho this PR is good to merge. maybe @mrmacete can give a 2nd opinion, but as it's a small step toward improving the arm analysis.

Reply:
i say that is wrong for sure because at 7418 there's no begining of function, it's in the middle of another function, this was caused by wrong thumb/arm code analysis.